champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/warlock/axe/Clone.java

```diff
@@ -253,6 +256,26 @@ public void onCustomDamageEvent(CustomDamageEvent event) {
         }
     }
 
+    @EventHandler
+    public void onEntityPropertyEvent(GetEntityRelationshipEvent event) {
+        if (!(event.getTarget() instanceof Vindicator vindicator)) return;
+
+        final Player owner = getCloneOwner(vindicator);
+        if (!clones.containsKey(owner)) return;
+        event.setEntityProperty(UtilEntity.getRelation(event.getEntity(), owner));
+    }
+
+    @EventHandler
+    public void onFetchNearbyEntityEvent(FetchNearbyEntityEvent<LivingEntity> event) {
+        event.getEntities().forEach(keyValue -> {
+            if (!(keyValue.getKey() instanceof Vindicator vindicator)) return;
+
+            final Player owner = getCloneOwner(vindicator);
+            if (!clones.containsKey(owner)) return;
+            keyValue.setValue(UtilEntity.getRelation(event.getSource(), owner));
+        });
+    }
+
```

Comment:
I think its better to solve this at the root. If you've rebased I added the PlayerSpawned metadata to the clone, I think if we check for that in the Clone customdamageevent and skip it will be better and apply to all future spawned entities